cpp/Piggybank/README.md

```diff
@@ -1,64 +1,16 @@
 # Piggybank
 
-## Methods
-This contract has three methods.
-* `Piggybank::constructor(MsgAddress owner, uint256_t limit)` - initialization of contract.
-* `Piggybank::deposit()` - receive transfer and increase internal balance.
-* `Piggybank::withdraw()` - check that sender is owner, balance is greater than limit and transfer all balance to owner.
-
-## Persistent data
-* MsgAddress `owner` - contract's owner, initialized in `Piggybank::constructor()` call.
-* uint256_t `limit` - limit to allow withdrawal.
-* uint256_t `balance` - current balance.
-
-## Compilation (manual, step-by-step, without tvm-build++)
```

Comment:
My plan is to describe common compilation / deploying workflow in the root of the cpp examples directory.